### PR TITLE
Hw3

### PR DESCRIPTION
**Pull request by chanjessica**

### CODE REVIEW

**Comment by chanjessica** [comment]
REVIEWERS is not availbe

File: lesson03/assignment/css/_leftCard.scss [hunk 1]
```diff
@@ -0,0 +1,33 @@
+// for picture on the left side
+
+.pic {
+	@include border-radius($radius);
+	/* border: 10px solid green;
+		overflow: auto;  */
+	 width: 230px;
+	 height: 200px;
```
Comment (brycebenson) [inline lesson03/assignment/css/_leftCard.scss:8]: The height here is causing issues in mobile, you'd be better suited to leave container height set to auto and control the elements within to dictate the height via content.

File: lesson03/assignment/index2.js [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+          "Experience": "20 years",
+          "School": "UNC",
+          "Major": "Trash Talking",
+          "Email": "mJordan@bulls.com",
+          "LinkedInUrl": "mJordas.linkedinprofile.com"
+        }];
+
+    let html="";
+    // Create the Person class constructor
+    class Person {
```
Comment (brycebenson) [inline lesson03/assignment/index2.js:44]: Good use of constructor and creating the template with a method.

File: lesson03/assignment/index2.js [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+                        </ul>
+                    </div>
+                </div>
+            </div>` ;
+            //insert template data into html document
+            //$(".template-hook'").append(html);
+            document.getElementById('template-hook').innerHTML = html;
+        }
+    }
+    let people=[];  //  consolthis.log("type of " + people);
```
Comment (brycebenson) [inline lesson03/assignment/index2.js:82]: This imperative approach works for your purposes

File: lesson03/assignment/css/_rightCard.scss [hunk 4]
```diff
@@ -0,0 +1,31 @@
+.desc {
+   /* border:  10px solid green;*/
+   width: 250px;
+   height: 200px;
```
Comment (brycebenson) [inline lesson03/assignment/css/_rightCard.scss:4]: Same idea with the declared height here.

**Review by brycebenson — APPROVED** [review]
Great job on this Jessica.

**Pull request merged**